code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -62,6 +63,11 @@ var/global/list/datum/stack_recipe/bananium_recipes = list ( \
 	new/datum/stack_recipe("bananium grenade casing", /obj/item/weapon/grenade/bananade/casing, 4, on_floor = 1), \
 	)
 
+var/global/list/datum/stack_recipe/mime_recipes = list ( \
+	new/datum/stack_recipe("silent tile", /obj/item/stack/tile/silent, 1, 4, 20), \
+	new/datum/stack_recipe("invisible wall", /obj/structure/barricade/mime, 10, one_per_turf = 1, on_floor = 1, time = 50), \

```

Comment:
To clarify a) they actually look like standing water, not completely invisible, b) you can crowbar them apart, c) At costing 10 sheets, when there's only 30 in existence, it's possibly the worst waste of the material.